ChangeLog

```diff
@@ -31,6 +31,10 @@ Changes
      Inserted as an enhancement for #1371
    * Add support for alternative CSR headers, as used by Microsoft and defined
      in RFC 7468. Found by Michael Ernst. Fixes #767.
+   * Remove dead code from bignum.c in the default configuration.
+     Found by Coverity, reported and fixed by Peter Kolbus. Fixes #2309.
```

Comment:
Please credit my employer as well: "Peter Kolbus (Garmin)"

Reply:
Done. Thanks for a quick response!